build/networking/scripts/build_p4c.sh

```diff
@@ -33,15 +33,8 @@ echo ""
 git clone https://github.com/p4lang/p4c.git --recursive P4C
 cd P4C
 git checkout $P4C_SHA
-#In protobuf version 3.18.1,'OK’ is no longer a member of
-#‘google::protobuf::util::status_internal::Status’. Instead a
-#public method (ok()) is available to check the return status.
-#This patch applies necessary changes to P4C code to accomodate
-#this change
-#TODO: See if this patch can be upstreamed to p4lang
-git apply $PATCH_DIR/PATCH-01-P4C
 mkdir build && cd build
-cmake -DENABLE_BMV2=OFF \ -DENABLE_P4C_GRAPHS=OFF -DENABLE_P4TEST=OFF \
```

Comment:
dpdk backend depends on some bmv2 passes, so we can not exclude from build

Reply:
We should honestly just stop disabling anything here and build with everything enabled.

Reply:
done

Reply:
I'm not sure that the dpdk backend depends on for bmv2 compiled code. i.e. the dpdk backend uses BMV2 passes (code) but should compile that code when creating the p4c-dpdk binary, it should not depend on code compiled for the p4c-bmv2 binary. If that is the case then we should create a issue in the p4c repo!